packages/react/src/index.ts

```diff
@@ -5,7 +5,11 @@ export { Profiler, withProfiler, useProfiler } from './profiler';
 export type { ErrorBoundaryProps, FallbackRender } from './errorboundary';
 export { ErrorBoundary, withErrorBoundary } from './errorboundary';
 export { createReduxEnhancer } from './redux';
-export { reactRouterV3Instrumentation } from './reactrouterv3';
+export {
+  // eslint-disable-next-line deprecation/deprecation
+  reactRouterV3Instrumentation,
+  browserTracingReactRouterV3Integration,
```

Comment:
Same as in https://github.com/getsentry/sentry-javascript/pull/10488, `reactRouterV3BrowserTracingIntegration` would sound a bit cleaner to me.